src/constants.py

```diff
@@ -25,11 +26,12 @@
 CARD_SIZE = INIT_CARD_WIDTH, INIT_CARD_HEIGHT = 69,100
 WIDTH_HEIGHT_RATIO = 500/726
 HORIZONTAL_MARGIN = 30
-BACKGROUND = pygame.image.load("./Assets/Fabric.jpg")
-BACKGROUND = pygame.transform.scale(BACKGROUND, SIZE)
-BACK = pygame.image.load("./Assets/Cards/back.png")
+BACKGROUND = pygame.image.load(ASSETS + "fabric.jpg")
+BACKGROUND = pygame.transform.smoothscale(BACKGROUND, SIZE)
+ICON = pygame.image.load(ASSETS + "cards/king_of_spades.png")
+BACK = pygame.image.load(ASSETS + "cards/back.png")
 ARROW_SIZE = ARROW_WIDTH, ARROW_HEIGHT = (25,15)
```

Comment:
```suggestion
ARROW_DIMENSIONS = ARROW_WIDTH, ARROW_HEIGHT = (25,15)
```
`SIZE `is a bit misleading